app/src/test/java/org/mozilla/fenix/settings/quicksettings/WebsiteInfoViewTest.kt

```diff
@@ -60,4 +64,56 @@ class WebsiteInfoViewTest {
 
         assertEquals("Insecure Connection", view.securityInfo.text)
     }
+
+    @Test
+    fun `WHEN updating on detailed mode THEN bind the certificate, title and back button listener`() {
+        val view = spyk(WebsiteInfoView(FrameLayout(testContext), icons, interactor, isDetailsMode = true))
```

Comment:
Is the spyk needed?

Reply:
Yes, see from lines 79 to 83.
<img width="1043" alt="image" src="https://user-images.githubusercontent.com/773158/125350178-65263100-e32c-11eb-9fea-fcee05ce6a69.png">